package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint",
-  "version": "2.0.0-rc.0",
+  "version": "2.0.0-beta.1",
```

Comment:
I think you need to rebase your branch with origin/master as this change is not correct.